### ISSUE TITLE
placesCenter@scollins: Panel Text Doesn't Show Anymore

### ISSUE THREAD

**Issue by SerafDosSantos** [linuxmint/cinnamon-spices-applets#249]
@collinss I think some of the late changes affects the way the Places Center applet behaves.

I don't see the panel text anymore. I have it in an horizontal panel which should display the panel text.

I've only updated the applet yesterday so I don't know since when this problem arose but it did since I updated.

**Comment by collinss** [comment]
It seems to work fine for me. What version of Cinnamon are you using?

**Comment by collinss** [comment]
Also, could you pastebin the contents of ```glass.log``` and ```.xsession-errors```?

**Comment by SerafDosSantos** [comment]
Cinnamon 3.2.7

glass.log: http://pastebin.com/0dLN424q
.xsession-errors: http://pastebin.com/YtqaZnpS

**Comment by collinss** [comment]
Hmm... looks like your logs are much more busy than they should be, but I don't see anything there that might be related to that issue. I can't reproduce the issue either, so I'm not sure how to find the problem, much less fix it. Could you try to change the text in the settings and see if anything related pops up in the logs? You can monitor them with

    tail -f .xsession-errors

and

    tail -f glass.log

(in separate terminals)

**Comment by SerafDosSantos** [comment]
I don't get much. I'll live without text for the moment. Let's keep this issue open for a while to see if it happens to others.

**Comment by mohammad-sn** [comment]
Hi @collinss, I noticed that it shows the text when applet is placed on the right or top panel, but hides it on left and bottom panels. At first I just had tested right and bottom panels so I supposed that true/false parameters for hide_applet_label function were misplaced. When I changed it and moved applet to the left panel to test it, I noticed the [erroneous](https://stackoverflow.com/questions/3067072/why-does-javascripts-in-operator-return-true-when-testing-if-0-exists-in-an-a) usage of the **in** operator.
